packages/widget-chat/src/components/activity-item-post/index.js

```diff
@@ -2,22 +2,55 @@ import React, {PropTypes} from 'react';
 import classNames from 'classnames';
 
 import Avatar from '../avatar';
-import ActivityItemPostActions from '../activity-item-post-actions';
+import ActivityItemPostAction from '../activity-item-post-action';
 import styles from './styles.css';
 
+import {ICON_TYPE_DELETE, ICON_TYPE_FLAGGED_OUTLINE} from '../icon';
 
 export default function ActivityItemPost(props) {
+  function handleOnDelete() {
+    const {id, onActivityDelete} = props;
+    onActivityDelete(id);
+  }
+
+  function handleOnFlag() {
+    const {id, onActivityFlag} = props;
+    onActivityFlag(id);
+  }
+
+  function wrapActionItem(action, highlight) { // eslint-disable-line react/no-multi-comp
```

Comment:
We may want to add the `ignoreStateless` rule for this one. Unless there's a compelling way to pull this out into its own file, like an `action-item-base` or something like that?